modules/flowable-engine-common/src/test/java/org/flowable/common/engine/impl/ServiceConfiguratorTest.java

```diff
@@ -0,0 +1,140 @@
+package org.flowable.common.engine.impl;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
We try to use AssertJ for our assertions. Can we perhaps adapt this?

Reply:
Certainly.